src/JustSaying/JustSaying.csproj

```diff
@@ -9,7 +9,7 @@
     <PackageReference Include="AWSSDK.SimpleNotificationService" Version="3.3.100.1" />
     <PackageReference Include="AWSSDK.SQS" Version="3.3.100.1" />
     <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="1.1.0" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="2.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="3.1.2" />
```

Comment:
Revert. This would block apps on 2.1 LTS from using JustSaying.

Reply:
But this version targets netstandard2.0 which 2.1 LTS supports?

Reply:
It's part of the 3.x major release, and to be honest I don't know what the consequences would be.

It's a library of interfaces, so there's no need to update it to the latest version in a library. If we leave it as it is, then there's no potential for issues.